### DISCUSSION TITLE
New updates coming

### DISCUSSION THREAD

**Discussion by mihir-bombay-studio** [dear-digital/linter#93] [Announcements]
Hi everyone,

I'm working on new updates this week on how we work together, I'll be preparing for that the next couple of days and after that create new issues. Thank you for your patience. 

PS: Those who haven't yet signed the internship offer letter, please do so or else you won't be able to send invoice.

**Comment by kmalap05** [comment]
I apologize for the delay, but I have indeed signed the Internship Offer Letter and sent it to you via mail.
